docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -186,6 +186,47 @@ Module not found: Can't resolve 'aws-crt' in 'xxx'
 
 These errors will not affect the use and functionality of the client, nor will errors appear in the browser console. This is a known transitive issue when using the AWS SDK. The issue is currently tracked [here](https://github.com/aws-amplify/amplify-js/issues/11030).
 
+## Server-Side Rendering (SSR)
+
+### Attempted import error: 'useForm' is not exported from 'react-hook-form' (imported as 'useForm').
+
+If you attempt to use Amplify with server-side rendering, you may encounter the following error:
+
+```
+./node_modules/@aws-amplify/ui-react-core/dist/esm/components/FormCore/FormProvider.mjs
+Attempted import error: 'useForm' is not exported from 'react-hook-form' (imported as 'useForm').
+```
+
+Amplify UI components are interactive and designed to work on the client side. To use them inside of Server Components you must wrap them in a Client Component with "use client". For more information, see the Next.js [documentation](https://nextjs.org/docs/app/building-your-application/rendering/composition-patterns#using-third-party-packages-and-providers) on how you could achieve this.
+
+## Vite
```

Comment:
Is this section moved up just for alphabetization?

Reply:
Yes lol it was annoying me